docs/ref/templates/builtins.txt

```diff
@@ -212,6 +212,22 @@ This tag can be used in two ways:
 
 See :ref:`template-inheritance` for more information.
 
+Literal form (with quotes) can use relative path (started from ``./`` or ``../``).
+
+Extend "base.html":
+
+* ``{% extends "./base.html" %}`` located in the same folder,
+  where your template placed
```

Comment:
English wash:

* * ``{% extends "./base.html" %}``: extend the "base.html" located in the same folder as the template

Comment: what happens if there is no "./base.html" but "base.html" would have worked?